client/galaxy/scripts/mvc/grid/grid-view.js

```diff
@@ -20,6 +20,7 @@ return Backbone.View.extend({
     initialize: function(grid_config) {
         this.grid = new GridModel();
         this.dict_format = grid_config.dict_format;
+        this.title = grid_config.title;
```

Comment:
This doesn't actually work all the time because grid_config.title is shipped *from* many of the dict endpoints, which get initialized and rendered async.  Need to look into a more robust solution.

Reply:
(handled this for now by setting a delayed title initialization flag indicating to views that the title hasn't been set yet and that they can set the window title on their own)